python/lsst/ts/salobj/domain.py

```diff
@@ -43,6 +43,55 @@
 MAX_RANDOM_HOST = (1 << 31) - 1
```

Comment:
The docstring for `DDS_READ_QUEUE_LEN` above says:

```
his must be equal to or greater than the queue length in the
OpenSplice configuration file (pointed to by $OSPL_URI).
```

But it is not in `OSPL_URI` right? Isn't it in the QoS file?

Also, is it still true that you can't read it from the QoS objects you create bellow? If you can read it from these objects created from the QoS file, then you can probably ditch this value here.

Reply:
Yes it is the QoS file. I will fix the doc string.

It is still true that I cannot read any QoS values from the dds library. It is maddening. But I don't understand why I can ditch this value here.

Reply:
Oh, I think I see your point. You mean that I should not have to set read_queue_policy:
```
        read_queue_policy = dds.HistoryQosPolicy(
            depth=DDS_READ_QUEUE_LEN, kind=dds.DDSHistoryKind.KEEP_LAST
        )
        self.reader_qos = self.qos_provider.get_reader_qos()
        self.reader_qos.set_policies([read_queue_policy])
```
I will try removing that code.

Reply:
I was able to remove it and all tests pass. Thank you for suggesting this change. Now *all* QoS is defined by the new QoS.xml file in ts_idl.